sql/moz-fx-data-shared-prod/monitoring/bigquery_usage_costs/view.sql

```diff
@@ -0,0 +1,32 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.monitoring.bigquery_usage_costs`
+AS
+SELECT
+  source_project,
+  user_email,
+  job_id,
+  SUM(total_terabytes_processed) * 4.15 AS cost_usd
+FROM
+  (
+    SELECT
+      source_project,
+      user_email,
+      job_id,
+      total_terabytes_processed
+    FROM
+      `moz-fx-data-shared-prod.monitoring.bigquery_usage`
+    WHERE
+      state = 'DONE'
+      AND job_type = 'QUERY'
+      AND reservation_id IS NULL
```

Comment:
For context, what's the value of `reservation_id` if it's filtering only NULLs?  Do we want to exclude jobs related to certain processes e.g. `reservation_id` = default-pipeline?

Reply:
I don't have enough context on reservations to know which filter is appropriate.  The ask was to mimic [billing.bigquery_usage_by_user](https://github.com/mozilla-services/dataops/blob/main/bigquery/billing/billing.bigquery_usage_by_user.sql) and use the same filtering.